volatility3/framework/objects/__init__.py

```diff
@@ -410,6 +410,19 @@ def _unmarshall(
         value = int.from_bytes(data, byteorder=endian, signed=signed)
         return value & mask
 
+    def get_raw_value(self) -> int:
+        formats = {
+            4: "I",
+            8: "Q",
+        }
+        length = self.vol.data_format.length
+        endian = self.vol.data_format.byteorder
+        raw_data = self._context.layers[self.vol.layer_name].read(
+            self.vol.offset, length
+        )
+        struct_format = ("<" if endian == "little" else ">") + formats[length]
+        return struct.unpack(struct_format, raw_data)[0]
```

Comment:
I think it would be better to use `int.from_bytes` so that it's consistent with the `unmarshall` call above.  You could probably just split the unmarshall above into two functions and expose the `get_raw_value` one...